mubench.reviewsite/phpunit.xml

```diff
@@ -0,0 +1,13 @@
+<phpunit bootstrap="vendor/autoload.php"
+         defaultTestSuite="unit_tests">
+    <testsuites>
+        <testsuite name="integration_tests">
+            <directory>tests/selenium</directory>
```

Comment:
If you place the Selenium tests below the unit tests, doesn't IntelliJ still try to execute them when running "all" tests?